src/main/scala/nl.knaw.dans.easy.managedeposit/Command.scala

```diff
@@ -51,9 +52,9 @@ object Command extends App with DebugEnhancedLogging {
     case commandLine.reportCmd :: (error @ commandLine.reportCmd.errorCmd) :: Nil =>
       app.createErrorReport(error.depositor.toOption, error.age.toOption)
     case (clean @ commandLine.cleanCmd) :: Nil =>
-      Console.out.println(s"Deleting ${ if(clean.dataOnly()) "data from " else "" }deposits with state ${clean.state()} for ${clean.depositor.toOption.getOrElse("all users")}")
-      if (cleanInteraction)
-        app.cleanDepositor(clean.depositor.toOption, clean.keep(), clean.state(), clean.dataOnly())
+      Console.out.println(s"${ if(clean.doUpdate()) "Deleting" else "To be deleted" } ${ if(clean.dataOnly()) "data from " else "" }deposits with state ${clean.state()}${ if(clean.newStateLabel.isSupplied) ", replacing with state "  else ""}${clean.newStateLabel.getOrElse("")} for ${clean.depositor.toOption.getOrElse("all users")}")
+      if (clean.force() || cleanInteraction())
+        app.cleanDepositor(clean.depositor.toOption, clean.keep(), clean.state(), clean.dataOnly(), clean.doUpdate(), clean.newStateLabel, clean.newStateDescription, clean.output())
```

Comment:
not sure, but shouldn't this be 

```suggestion
        app.cleanDepositor(clean.depositor.toOption, clean.keep(), clean.state(), clean.dataOnly(), clean.doUpdate(), clean.newStateLabel.toOption, clean.newStateDescription.toOption, clean.output())
```

Reply:
yep